x-pack/plugins/security_solution/public/common/components/super_date_picker/selectors.ts

```diff
@@ -48,4 +59,7 @@ export const queriesSelector = () =>
   createSelector(getGlobalQueries, (queries) => queries.filter((q) => q.id !== 'kql'));
 
 export const kqlQuerySelector = () =>
-  createSelector(getQueries, (queries) => queries.find((q) => q.id === 'kql'));
+  createSelector(
+    getQueries,
+    (queries) => queries.find((q) => q.id === 'kql') as unknown as GlobalKqlQuery | undefined
```

Comment:
cannot get this to work any other way, open to suggestions

Reply:
Can we just update this? https://github.com/elastic/kibana/blob/eee492ab7e848c93597ee6166896efaa2b400b52/x-pack/plugins/security_solution/public/common/store/inputs/model.ts#L76

Pretty sure we're not using GraphQL anymore. So we can just delete GlobalQuery and use GlobalKQLQuery everywhere?

Reply:
i did that initially, but led to a lot of file changes for fixing inconsistent types with the `refetch` property. I will do this as part of the type follow up PR.